.github/workflows/image-build-binary.yml

```diff
@@ -57,34 +61,81 @@ jobs:
         run: |
           echo "docker-org=${{ github.repository_owner }}" >> "$GITHUB_ENV"
 
+      - name: Check Signer Release
+        id: check_signer_release
+        run: |
+          case "${{ inputs.tag }}" in
+            signer-*)
+              echo "is-signer-release=true" >> $GITHUB_ENV
+              ;;
+            *)
+              echo "is-signer-release=false" >> $GITHUB_ENV
+              ;;
+          esac
+
       ## Set docker metatdata
       ## - depending on the matrix.dist, different tags will be enabled
       ## ex. debian will have this tag: `type=ref,event=tag,enable=${{ matrix.dist == 'debian' }}`
       - name: Docker Metadata ( ${{matrix.dist}} )
-        id: docker_metadata
+        if: ${{ env.is-signer-release == 'true' }}
+        id: docker_metadata_signer
+        uses: docker/metadata-action@8e5442c4ef9f78752691e2d8f8d19755c6f78e81 #v5.5.1
+        with:
+          images: |
+            ${{env.docker-org}}/stacks-signer
+          tags: |
+            type=raw,value=latest,enable=${{ inputs.docker_tag != '' && (github.ref == format('refs/heads/{0}', github.event.repository.default_branch) ) && matrix.dist == 'debian' }}
+            type=raw,value=${{ inputs.docker_tag }}-${{ matrix.dist }},enable=${{ inputs.docker_tag != '' && matrix.dist == 'debian'}}
+            type=raw,value=${{ inputs.docker_tag }},enable=${{ inputs.docker_tag != '' && matrix.dist == 'debian' }}
+            type=ref,event=tag,enable=${{ matrix.dist == 'debian' }}
+            type=raw,value=latest-${{ matrix.dist }},enable=${{ inputs.docker_tag != '' && (github.ref == format('refs/heads/{0}', github.event.repository.default_branch) ) && matrix.dist == 'alpine' }}
+            type=raw,value=${{ inputs.docker_tag }}-${{ matrix.dist }},enable=${{ inputs.docker_tag != '' && matrix.dist == 'alpine' }}
```

Comment:
I think the last tag:
```
type=raw,value=${{ inputs.docker_tag }}-${{ matrix.dist }},enable=${{ inputs.docker_tag != '' && matrix.dist == 'alpine' }}
```

may be a duplicate of the second tag on [line 88](https://github.com/stacks-network/stacks-core/pull/4968/files#diff-8162b8d1ab123855ca5811df021e82d683361ffab6b5104865d12c85541cfc4cR88):
```
type=raw,value=${{ inputs.docker_tag }}-${{ matrix.dist }},enable=${{ inputs.docker_tag != '' && matrix.dist == 'debian'}}
```

 and can be simplified slightly by removing the last line and removing the final condition from line 88 like so:
```suggestion
            type=raw,value=latest,enable=${{ inputs.docker_tag != '' && (github.ref == format('refs/heads/{0}', github.event.repository.default_branch) ) && matrix.dist == 'debian' }}
            type=raw,value=${{ inputs.docker_tag }}-${{ matrix.dist }},enable=${{ inputs.docker_tag != ''}}
            type=raw,value=${{ inputs.docker_tag }},enable=${{ inputs.docker_tag != '' && matrix.dist == 'debian' }}
            type=ref,event=tag,enable=${{ matrix.dist == 'debian' }}
            type=raw,value=latest-${{ matrix.dist }},enable=${{ inputs.docker_tag != '' && (github.ref == format('refs/heads/{0}', github.event.repository.default_branch) ) && matrix.dist == 'alpine' }}
```

Reply:
This is similar to another issue we noticed that manifested around 2.5.0.0.0 in fact - since we stopped building exclusively from `default_branch`, the tags are a bit off. 
we still tag correctly, but for example - we're no longer publishing a `latest` tag when a new release is built.

we're looking into it, and will check the above as part of that work. ping @BowTiedDevOps